assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -123,10 +123,10 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
           parse: false,
         });
 
-        // TODO add headers for totals by status and have header reflect search
-        // only update totals when data is different
-        const totalStories = parseInt(response.headers.get('X-WP-Total'));
         const totalPages = parseInt(response.headers.get('X-WP-TotalPages'));
+        const totalStoriesByStatus = JSON.parse(
```

Comment:
I would check this exists before using it. Just incase.

Reply:
good idea, i'll add it to other data we're grabbing from headers too